concent_api/core/tests/test_auth_accept_or_reject_results.py

```diff
@@ -48,13 +48,13 @@ def test_provider_forces_subtask_results_for_task_which_was_already_submitted_co
         # STEP 1: Provider forces subtask results via Concent.
         # Request is processed correctly.
         serialized_force_subtask_results = self._get_serialized_force_subtask_results(
-            timestamp                   = "2018-02-05 10:00:15",
+            timestamp                   = "2018-02-05 10:00:25",
```

Comment:
Could you make this value '30' like in all other tests with this step ? It would be easier to find similar pieces of code in the future.

Reply:
Done